generators/client/files-react.js

```diff
@@ -51,6 +52,10 @@ const files = {
         {
             condition: generator => !generator.skipCommitHook,
             templates: ['.huskyrc']
+        },
+        {
+            condition: generator => generator.websocket === 'spring-websocket',
```

Comment:
Why does spring-websocket require TSLint?

Reply:
because we use the following `rxjs` linting rules that are not yet available in `eslint`. If we agree to drop these rules, then, `tslint` dependency can be dropped from the react code.

>     "rxjs-collapse-imports": true,
>     "rxjs-pipeable-operators-only": true,
>     "rxjs-no-static-observable-methods": true,
>     "rxjs-proper-imports": true
>

Reply:
Lets drop them, they are not worth branching out

Reply:
sure. will update to remove `rxjs` linting rules. 

Also, what's your preference on import/export rules like below. These are not well supported by current code and I have to exclude at multiple places specifically in protractor and barrels. BTW, we don't have such rules on `angular` front. If we exclude these, then, one more less dependency to maintain :smile:

>     "import/no-internal-modules": [ "error", {
>         "allow": ["app", "test", "uuid/v4", "@fortawesome/**", "lodash/**", "react-toastify/**"]
>       }],
>     "import/no-extraneous-dependencies": [ "error", {
>         "devDependencies": true
>       }],
>     "import/no-unresolved": "off"

Reply:
Feel free to modify them as you see fit if it helps with maintainability.

Reply:
PR updated. @mraible, @deepu105 do you have further suggestions?